cmd/content.go

```diff
@@ -177,9 +178,32 @@ func runContentCmd(cmd *cobra.Command, args []string) (err error) {
 		return errors.Wrap(err, "CommitOnBranchV4")
 	}
 
-	if title := viper.GetString("pr-title"); newBranch && title != "" {
+	if title := viper.GetString("pr-title"); title != "" {
+		existingPR, err := client.GetOpenPullRequestV4(owner, repo, branch, baseBranch)
+		if err != nil {
+			return errors.Wrap(err, "GetOpenPullRequestV4")
+		}
+
+		if existingPR != nil {
+			log.Infof("existing pull request found: %s", existingPR.URL)
+			return nil
+		}
+
+		if title := viper.GetString("pr-title"); (newBranch || (existingPR == nil)) && title != "" {
```

Comment:
You already have `title` set, no need to repeat this test :-)

Reply:
yes not sure how that snuck through cleaned up